examples/SinusoidInterface/README.md

```diff
@@ -0,0 +1,47 @@
+## Running this example.
+
+First instantiate the example e.g.,
+```
+cd examples/SinusoidInterface
+julia --project
+]
+instantiate
+```
+Then run the bash script
+```
+bash calibrate_script
+```
+The prior distributions are stored in `parameters.toml` in toml format. More examples of defining parameter distributions in this way are given in `test/TOMLInterface/toml/`.
+
+After it runs, (it takes *several minutes* to run, see below...), results are stored in `output/eki.jld2` to view results, load up with `julia --project` as follows
+```julia
+julia> using EnsembleKalmanProcesses, JLD2
+
+julia> @load "output/eki.jld2"
+3-element Vector{Symbol}:
+ :eki
+ :param_dict
+ :prior
+```
+Then, for example the final 5-member parameter ensemble is given by:
+```julia
+julia> get_ϕ_final(prior,eki)
+2×5 Matrix{Float64}:
+ 1.31749  1.28875  1.27677  1.33311  1.3305
+ 6.27765  6.63572  6.65148  6.66096  6.56518
+```
+
+## Why run this example in this way?
+
+The purpose of this example is to demonstrate how the EKP update and the model file are **completely disconnected** (e.g. the model can be written in another language, can require HPC and any data-processing tools). This calibration is called from a simple script-language (bash) and the interaction between the calibration tools and the forward map are only through simple readable files.
+
+- When ensembles are created they generate an ensemble of parameter directories, each containing a file in a universal TOML format
+`output/iteration_X/member_Y/parameters.toml`
+- When each `run_computer_code.jl` is run, it reads from one specific parameter file given from the pair `(X,Y)`, and outputs data into this directory (stored in JLD2, but could be any format)
```

Comment:
```suggestion
- Each `run_computer_code.jl` reads from one specific parameter file given from the pair `(X,Y)`, and outputs data into this directory (stored in JLD2, but could be any format)
```

Reply:
Do you define what the pair (X,Y) denotes?

Reply:
They come from the file-names in the line above